database/migrations/2023_10_04_094248_create_roles_table.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('roles', function (Blueprint $table) {
+            $table->id();
+            $table->string('name');
+            $table->timestamps();
```

Comment:
You can also add a `label` and `description` columns for better UI representation.

Reply:
I could add label as a slug if it's OK. Name could be Super Admin, and label super-admin. Is that what you were thinking? And same for permissions

Reply:
`super_admin` could be the name, `Super Admin` could be the label.

Reply:
Ok

Reply:
Done.